test-app/package.json

```diff
@@ -27,14 +28,17 @@
     "@ember/optional-features": "2.0.0",
     "@ember/string": "3.1.1",
     "@ember/test-helpers": "2.9.3",
-    "@embroider/macros": "1.13.2",
+    "@embroider/macros": "1.13.0",
     "@embroider/test-setup": "3.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@glint/core": "^1.2.0",
     "@glint/environment-ember-loose": "^1.2.0",
     "@glint/template": "^1.2.0",
     "@tsconfig/ember": "^3.0.1",
+    "@types/ember__component": "^4.0.18",
+    "@types/ember__object": "^4.0.8",
```

Comment:
Do we need this? Since we are using the preview types from Ember, I think we shouldn't

Reply:
It could be needed but only for ember-source < 4.8, so we could add these dependencies to `emberTypesPackages` in `ember-try.js`